src/nft/components/explore/Banner.tsx

```diff
@@ -115,21 +115,13 @@ const TRENDING_COLLECTION_SIZE = 5
 const Banner = () => {
   const navigate = useNavigate()
 
-  const { data } = useQuery(
-    ['trendingCollections'],
-    () => {
-      return fetchTrendingCollections({
-        volumeType: 'eth',
-        timePeriod: TimePeriod.OneDay,
-        size: TRENDING_COLLECTION_SIZE + EXCLUDED_COLLECTIONS.length,
-      })
-    },
-    {
-      refetchOnReconnect: false,
```

Comment:
btw I'm not sure if I should be doing this, but I just noticed nft collection page and banner was also not refetching prices - was that deliberate or nah?